infra/recipes/terraform/azure/modules/image/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "docker_image" "housetables" {
+    name = "${var.registry_login_server}/openhouse-housetables-service:latest"
+
+    build {
+        context = "${path.cwd}/../../../../../.."
```

Comment:
Why the path is in this format? Is it not possible to specify path from the root?

Reply:
Good point. I can just use the root.

Reply:
That still only returns the root of the terraform project though, so will still require some `..`